RELEASES.md

```diff
@@ -57,7 +57,21 @@ When the release branch is ready, a tag should be pushed with a name matching th
 
 ### Weekly Dev Release
 
-Ratify is configured to generate and publish dev build images based on schedule [here](https://github.com/deislabs/ratify/blob/main/.github/workflows/publish-package.yml#L8).  
+#### Publishing Guidelines
+- Ratify is configured to generate and publish dev build images based on the schedule [here](https://github.com/deislabs/ratify/blob/main/.github/workflows/publish-package.yml#L8). 
+- Contributors MUST select the `Helm Chart Change` option under the `Type of Change` section if there is ANY update to the helm chart that is required for proposed changes in PR.
+- Maintainers MUST manually trigger the "Publish Package" workflow after merging any PR that indicates `Helm Chart Change`
```

Comment:
wonder if we should add this to the pr template as well in case some maintainers might not know it.

Reply:
Sure I can add it there too